src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java

```diff
@@ -0,0 +1,54 @@
+package com.google.checkstyle.test.chapter4formatting.rule4854fieldannotations;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class InputFieldAnnotations {
+    public @interface SomeAnnotation1 { }
+
+    public @interface SomeAnnotation2 { }
+
+    public @interface SomeAnnotation3 {
+        int x();
+    }
+
+    /**
+     * testing.
+     */
+    @SomeAnnotation1
+    @SomeAnnotation2
+    String name = "Zops";
+
+    /**
+     * testing.
+     */
+    @SomeAnnotation1 @SomeAnnotation2
+    int age = 19;
```

Comment:
Please add a usecase with two annotations, one is on its own line and the second one is on the same line as the field, e.g.:
```
    /**
      * testing.
      */
     @SomeAnnotation1
     @SomeAnnotation2 int age = 19;
```

Reply:
done

https://github.com/checkstyle/checkstyle/blob/11ec1da697889f3b08cf7e2cd8f30b593dacc45b/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java#L28-L38